enocean/protocol/EEP.xml

```diff
@@ -1372,6 +1372,177 @@
           </enum>
         </data>
       </profile>
+      <profile type="0x04" description="Heating Radiator Valve Actuating Drive with feed and room temperature MEasurement, Local Set point and Display (BI-DIR)">
+        <data direction="1">
+          <value shortcut="CP" description="Current Value" offset="0" size="8" unit="%">
+            <range>
+              <min>0</min>
+              <max>100</max>
+            </range>
+            <scale>
+              <min>0</min>
+              <max>100</max>
+            </scale>
+          </value>
+          <value shortcut="FTS" description="Feed Temperature OR Temperature Set Point" offset="8" size="8" unit="°C">
+            <range>
+              <min>0</min>
+              <max>255</max>
+            </range>
+            <scale>
+              <min>10</min>
+              <max>30</max>
+            </scale>
+          </value>
+          <value shortcut="TMPFC" description="Room Temperature OR Failure Code" offset="16" size="8" unit="°C">
+            <range>
+              <min>0</min>
+              <max>255</max>
+            </range>
+            <scale>
+              <min>10</min>
+              <max>30</max>
+            </scale>
+          </value>
+          <enum shortcut="MST" description="Measure Status" offset="24" size="1">
+            <item description="active" value="0" />
+            <item description="inactive" value="1" />
+          </enum>
+          <enum shortcut="STR" description="Status Request" offset="25" size="1">
+            <item description="No change" value="0" />
+            <item description="status request" value="1" />
+          </enum>
+          <enum shortcut="LRNB" description="LRN Bit" offset="28" size="1">
```

Comment:
No need to declare the learn bit. This is automatically handled by the library.

Reply:
removed